healthcare/healthcare/doctype/patient_appointment/patient_appointment.py

```
@@ -631,6 +646,63 @@ def get_appointment_item(appointment_doc, item):
+	}
+
+
+def get_registration_fee_settings():
+	settings = {"include_registration_fee": 0, "reg_item": None, "reg_fee": 0}
+	try:
+		settings["include_registration_fee"] = cint(
+			frappe.db.get_single_value("Healthcare Settings", "include_registration_fee")
+		)
+		settings["reg_item"] = frappe.db.get_single_value("Healthcare Settings", "reg_item")
+		settings["reg_fee"] = flt(frappe.db.get_single_value("Healthcare Settings", "reg_fee"))
+	except Exception:
```

Ruff (0.15.4)

[warning] Do not catch blind exception: `Exception`

(BLE001)